Source/Core/Core/CoreParameter.h

```diff
@@ -185,6 +185,9 @@ struct SCoreStartupParameter
 	bool bDCBZOFF;
 	int iBBDumpPort;
 	bool bSyncGPU;
+	int iSyncGpuMaxDistance;
```

Comment:
What an excellent opportunity to start documenting obscure options while introducing them! ;)